bin/new-notification-config.js

```diff
@@ -0,0 +1,78 @@
+const path = require('path');
+const fs = require('fs-extra');
+const inquirer = require('inquirer');
+const chalk = require('chalk');
+
+const { creatingProjectSpinner } = require('./spinners');
+
+const setupNotificationConfig = async (notification) => {
+  const notificationType = getNotificationType(notification);
+  await fs.copy(
+    path.resolve(__dirname, `./../lib/notification/${notificationType.name}/`),
```

Comment:
This would change because of the request changes I made on this file - lib/notification/mailgun/wrapper.js

Reply:
Got it